.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -18,5 +18,5 @@
 - [ ] Deploy spell `ETH_GAS="XXX" ETH_GAS_PRICE="YYY" make deploy`
 - [ ] Verify `mainnet` contract on etherscan
 - [ ] Change test to use mainnet spell address and deploy timestamp
-- [ ] Keep `DssSpell.sol` and `DssSpell.t.sol` the same, but make a copy in `archive`
+- [ ] Run `make name="YYYY-MM-DD-archive-directory-name" archive-spell` to make an archive directory and copy `DssSpell.sol`, `DssSpell.t.sol`, and `DssSpell.t.base.sol`
```

Comment:
```suggestion
- [ ] Run `make name="YYYY-MM-DD-DssSpell" archive-spell` to make an archive directory and copy `DssSpell.sol`, `DssSpell.t.sol`, and `DssSpell.t.base.sol`
```

Reply:
in Goerli we seem to be using more descriptive names, do we want all the directories named DssSpell here or should we also start using more descriptive dir names?

Reply:
Good question.
I'm not sure as in spells-mainnet we followed the `DssSpell` pattern so far.
I personally would lean to keep it for consistency and readability, as you'll have an exec copy hashed in the spell that's not present for goerli, so this could explain the extra verbosity on goerli.

Reply:
Updated the scripts to use a date and add `-DssSpell` to the end